Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
> I'm tempted to suggest we not even use the makefile and run the commands inline so it's transparent what this actually does

This sounds good to me. You're definitely right in that the patch is fairly opaque, and the compilation steps are so simple in this case that they're hardly worth the added complexity of patching-then-running make.

@DomT4 thoughts?